aqt/editor.py

```diff
@@ -371,12 +371,15 @@ def showDupes(self):
                               contents))
         browser.onSearchActivated()
 
-    def fieldsAreBlank(self):
+    def fieldsAreBlank(self, previousNote=None):
         if not self.note:
             return True
         m = self.note.model()
         for c, f in enumerate(self.note.fields):
-            if f and not m['flds'][c]['sticky']:
+            notChangedvalues = {"", "<br>"}
+            if previousNote and m['flds'][c]['sticky']:
+                notChangedvalues.add(previousNote.fields[c])
+            if f not in notChangedvalues:
```

Comment:
Any particular reason you went with this approach instead of just comparing each field in turn? While unlikely, I guess it could be tricked if the contents of two fields is switched.

Also, if a user adds a field to the note type so that note.fields is larger than previousNote.fields, does that cause problems?

Reply:
No particular reason. It wast just the way it cames naturally to me when I did code it.

I indeed gave no thought about changing note type. I'll guess in this case, the safest thing to do would be to set previousNote to false for safety

Reply:
change done